app/src/test/java/com/nervousfish/nervousfish/BaseTest.java

```diff
@@ -0,0 +1,99 @@
+package com.nervousfish.nervousfish;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+/**
+ * The superclass of all tests
+ */
+public final class BaseTest {
+
+    /**
+     * Prevents instantiation
+     */
+    private BaseTest() {
+        // Prevents instantiation
+    }
+
+    /**
+     * Creates a new instance of a class that has an inaccessible constructor (eg private)
+     * Note that there should be a single constructor in the class
+     *
+     * @param clazz The class that should be instantiated
+     * @param args  The arguments that should be passed to the constructor
```

Comment:
Don't know, I just press Ctrl-Alt-Shift-L and then Intellij aligns everything to the official Java standard